panel/src/components/Drawers/FormDrawer.vue

```diff
@@ -80,10 +80,14 @@ export default {
 
 			if (focus !== false) {
 				setTimeout(() => {
-					let autofocus =
-						Object.values(this.fields).filter(
+					let autofocus = focus;
+
+					if (autofocus === true) {
```

Comment:
This looks a bit weird, we first check `focus` to not be `false` to then immediately check `autofocus` (which is just `focus`) to be `true`?